src/components/Dashboard/Notetaker/Category/CategoryItem.jsx

```diff
@@ -85,8 +92,19 @@ const CategoryItem = ({
             ) : (
                 <CategoryItemElementContainer>
                     <CategoryItemElement onClick={() => onPick(category)} isPicked={isPicked}>
+                        {showCheckbox && ( // Conditionally render the checkbox based on the showCheckbox prop
+                            <CategoriesSidebarCheckbox
+                                id={`checkbox-${id}`}
+                                onChange={(e) => {
+                                    e.target.checked ? dispatch(AddSelectedCategory(id)) : dispatch(RemoveCategory(id));
+                                }}
+                            />
+                        )}
+                        {showCheckbox && <label htmlFor={`checkbox-${id}`}></label>}{" "}
```

Comment:
Why do you need label when it is empty?

Reply:
For some reason it has to be explicitly defined given a value otherwise it wont render.

Reply:
ok, thanks